src/pages/job-apply/apply/mobile/mobile.tsx

```diff
@@ -10,8 +10,11 @@ import { ExpandableText } from 'src/components/atoms/expandable-text';
 import { printWhen } from 'src/core/utils';
 import { useApplyShared } from '../apply.shared';
```

Comment:
We are avoiding having different pages for mobile and desktop. Can you refactor your code to have 1 page for both mobile and desktop?pls checkout latest merge requests.

Reply:
I wasn't aware of this when I was developing. Can we accept this and refactor it later along with the other pages?